Agents/NTUPVLibAgent/README.md

```diff
@@ -0,0 +1,176 @@
+- `db.update.url` the [JDBC URL](https://www.postgresql.org/docs/7.4/jdbc-use.html) for the Postgres database to update to
+- `db.update.user` the username to access the Postgres database to update to
+- `db.update.password` the password to access the Postgres database to update to
+- `air.temperature.iri` the timeseries data IRI for air temperature
+- `wind.speed.iri` the timeseries data IRI for wind speed
+- `irradiance.iri` the timeseries data IRI for irradiance
+- `iri` the IRI of the sensor or weather station entity that has rdf:type s3n:SmartSensor or rdf:type ontoems:ReportingStation
+
+More information can be found in the property file `ts_client.properties` in the `resources` folder.
+
+
+## 1. PV Model Data Preparation
+####  [Option 1] Read photovoltaic model specs from property files
+
+This is the default option for the NTU test case.
+
+##### Model properties
+The model parameters properties contains the parameters required to create a solar PV Model for calculations. The file `model_parameters.properties` can be found in the `resources` folder. It should contain the following keys:
+- `temp_model` the temperature model (sapm or pvsyst)
+- `temp_model_config` the temperature model configuration (for sapm: open_rack_glass_glass or close_mount_glass_glass or open_rack_glass_polymer or insulated_back_glass_polymer, for pvsyst: freestanding or insulated)
+- `latitude` the latitude of the Solar PV Model location, can be left empty in which it will be queried from the knowledge graph
+- `longitude` the longitude of the Solar PV Model location, can be left empty in which it will be queried from the knowledge graph
+- `altitude` the altitude of the Solar PV Model location
+- `surface_tilt` the surface tilt angle of the Solar PV Model
+- `surface_azimuth` the surface azimuth of the Solar PV Model
+- `module_rated_dc_power` the rated DC power of the PV module
+- `module_temperature_coefficient` the temperature coefficient of the PV module
+- `inverter_rated_dc_power` the rated DC power of the inverter
+- `modules_per_string` the number of modules per string
+- `strings_per_inverter` the number of strings per inverter
+
+##### Misc properties
+
+The agent takes a scale factor that can be applied to the calculated power. The scale_factor can be set in the file `misc_parameters.properties` found in the `resources` folder.
+
+####  [Option 2] Read Photovoltaic Model Specs from Knowledge Graph
+Alternatively, the parameters required to create a solar PV Model can be read from a knowledge graph. This requires an instantiation agent to create a Knowledge Graph filled with PV model parameter values. The  [HistoricalNTUEnergy Agent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/HistoricalNTUEnergyAgent) provides an example to instantiate a knowledge graph which includes PV model parameters.
+
+This option is currently only available for latitude and longitude. 
```

Comment:
I don't understand this statement - is it that this option can be used with lat/lon as input? what are other possible inputs?